github/interactions_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
```

Comment:
```suggestion
// Copyright 2021 The go-github AUTHORS. All rights reserved.
```

Reply:
Sure @gmlewis. Will make the change.
Thank you for your instantaneous feedback.